package.json

```
@@ -1015,6 +1024,11 @@
                     "when": "view == aws.explorer && viewItem == awsS3FileNode",
                     "group": "0@1"
                 },
+                {
+                    "command": "aws.s3.openFile",
+                    "when": "view == aws.explorer && viewItem == awsS3FileNode",
+                    "group": "0@2"
```

why a different group?
```suggestion
                    "group": "0@1"
```

---

Isn't the second number just the order that it appears on?